src/riak_kv_qry_compiler.erl

```diff
@@ -630,11 +698,11 @@ simplest_test() ->
     PK = get_standard_pk(),
     LK = get_standard_lk(),
     ?assertMatch(
-        [#riak_sql_v1{ is_executable = true,
-                       type          = timeseries,
-                       'WHERE'       = Where2,
-                       partition_key = PK,
-                       local_key     = LK }],
+        {ok, _, [#riak_sql_v1{ is_executable = true,
```

Comment:
The `InitialState` here is `[]`, so I suppose you should be matching against that, rather than `_`.